types/chrome-webstore-upload/index.d.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference types="node" />
+
+export interface Options {
+    extensionId: string;
+    clientId: string;
+    clientSecret?: string;
+    refreshToken: string;
+}
+
+export interface Item {
+    kind: string;
+    id: string;
+    publicKey: string;
+    uploadState: "FAILURE" | "IN_PROGRESS" | "NOT_FOUND" | "SUCCESS";
+    itemError: {
+        error_code: string;
+        error_detail: string;
+    };
+}
+
+export type PublishStatus =
+    | "OK"
+    | "NOT_AUTHORIZED"
+    | "INVALID_DEVELOPER"
+    | "DEVELOPER_NO_OWNERSHIP"
+    | "DEVELOPER_SUSPENDED"
+    | "ITEM_NOT_FOUND"
+    | "ITEM_PENDING_REVIEW"
+    | "ITEM_TAKEN_DOWN"
+    | "PUBLISHER_SUSPENDED";
+
+export interface PublishResponse {
+    kind: string;
+    item_id: string;
+    status: PublishStatus[];
+    statusDetail: string[];
+}
+
+export default function chromeWebstoreUpload(options: Options): {
+    uploadExisting(readStream: NodeJS.ReadableStream, token?: string): Promise<Item>;
```

Comment:
I'd would probably:
- convert this to interface/type (APIClient)
- add `@async` JSDoc tag, as declarations files are missing support for 'async' keyword in declarations